src/Emulator/Peripherals/Peripherals/Miscellaneous/K6xF_SIM.cs

```diff
@@ -0,0 +1,111 @@
+﻿//
+// Copyright (c) 2010-2020 Antmicro
+//
+//  This file is licensed under the MIT License.
+//  Full license text is available in 'licenses/MIT.txt'.
+//
+using System;
+using System.Collections.Generic;
+using Antmicro.Renode.Core;
+using Antmicro.Renode.Core.Structure.Registers;
+using Antmicro.Renode.Logging;
+using Antmicro.Renode.Peripherals.Bus;
+
+namespace Antmicro.Renode.Peripherals.Miscellaneous
+{
+    public class K6xF_SIM : IDoubleWordPeripheral, IKnownSize
+    {
+        public K6xF_SIM()
+        {
+            var rng = EmulationManager.Instance.CurrentEmulation.RandomGenerator;
+            uniqueIdHigh = (uint)rng.Next();
```

Comment:
Should those values really be random?
An alternative solution would be to add constructor parameters and pass them in the `repl` file.

Reply:
Good point. How about to allow both? The parameters could take a default value of 0. For the default case, random values are generated and otherwise the parameter value is used?